src/components/loader-bar/loader-bar-test.stories.tsx

```diff
@@ -1,10 +1,13 @@
 import React from "react";
+import { ComponentStory } from "@storybook/react";
 import LoaderBar, { LoaderBarProps } from ".";
 import { LOADER_BAR_SIZES } from "./loader-bar.config";
+import Box from "../box";
+import Typography from "../typography";
 
 export default {
   title: "Loader Bar/Test",
-  includeStories: ["DefaultStory"],
+  includeStories: ["DefaultStory", "HeightFix"],
```

Comment:
**reminder:** This will need updating with the name you give the new story too. I forget all the time to update this and wonder why the example hasn't loaded in Storybook 😄